schema/me/sale_registrations.js

```diff
@@ -0,0 +1,45 @@
+import { first } from 'lodash';
+import gravity from '../../lib/loaders/gravity';
+import Sale from '../sale';
+import Sales from '../sales';
+import Bidder from '../bidder';
+import {
+  GraphQLList,
+  GraphQLBoolean,
+  GraphQLObjectType,
+} from 'graphql';
+
+export const SaleRegistrationType = new GraphQLObjectType({
+  name: 'SaleRegistration',
+  fields: () => ({
+    is_registered: {
+      type: GraphQLBoolean,
+    },
+    bidder: {
+      type: Bidder.type,
+    },
+    sale: {
+      type: Sale.type,
+    },
+  }),
+});
+
+export default {
+  type: new GraphQLList(SaleRegistrationType),
+  args: Sales.args,
+  resolve: (root, options, { rootValue: { accessToken } }) => {
+    return gravity('sales', options)
```

Comment:
This accepts the Sales.args. The query I'm using is: https://github.com/artsy/force/commit/be2534481c6393e27a7209d8e690ae4f72212126#diff-cb48918a0d6bf9cb4ffdf1df18da8ddeR4

`sale_registrations(published: true, is_auction: true, sort: CREATED_AT_DESC)`

It would be good to actually query using an array of `auction_states`—though I'm not holding my breath. For the /auctions page we just over-fetch and group whatever we get.